src/nameserver/name_server_impl.cc

```diff
@@ -9973,12 +9983,52 @@ void NameServerImpl::DropProcedureOnTablet(const std::string& db_name, const std
             tb_client_vec.push_back(kv.second->client_);
         }
     }
+
+    bool is_deployment_procedure = false;
+    {
+        std::lock_guard<std::mutex> lock(mu_);
+        auto it = db_sp_info_map_.find(db_name);
+        if (it != db_sp_info_map_.end()) {
+            is_deployment_procedure = it->second.find(sp_name) != it->second.end();
```

Comment:
check type in procedure info to make sure it is a deployment?

Reply:
uh right